### PR TITLE
Relational operators in Table implementation

### PR DESCRIPTION
**Pull request by icandigitbaby**
This PR implements different relational operations used in restrictions, such as `NotEqual`, `LessThan`, etc. It doesn't implement the DL membership operator. Comparisons are made in a Pythonic way and types are not checked.

The PR also includes a small corrections that uses the wrapping function for decoding data from the REST calls in FAI message retrieval.

### CODE REVIEW

**Comment by jkerihuel** [comment]
I am going to put this PR on hold because I have pending commits that are doing a similar thing

File: mapiproxy/libmapistore/backends/python/rest.py [hunk 1]
```diff
@@ -530,6 +528,19 @@ class PidTagAccessFlag(object):
     AssocContentsTable  = 0x00000020
 
 
+class RelOperator(object):
+    """ Define possible values for property comparison operators
```
Comment (jelmer) [inline mapiproxy/libmapistore/backends/python/rest.py]: Please use PEP8: one-line summary for docstrings, no leading or trailing whitespace.

File: mapiproxy/libmapistore/backends/python/rest.py [hunk 2]
```diff
@@ -530,6 +528,19 @@ class PidTagAccessFlag(object):
...
+        included in 'property' restriction structures """
+
+    LessThan            = 0x00
```
Comment (jelmer) [inline mapiproxy/libmapistore/backends/python/rest.py]: Please use one space between name and equals sign, per PEP8.

**Comment by icandigitbaby** [comment]
Relational operators were removed from the PR due to their implementation in a different one. The remaining commit is still valid, although the name of the PR doesn't apply anymore.

**Pull request merged**